docs/en/CSMM/advanced-feature-guide-deadlydrops.md

```diff
@@ -21,32 +21,14 @@ zcoord :    (?<=Spawned supply crate at \(-?\d{1,4}\.\d, -?\d{1,4}\.\d, )(-?\d{1
+We will use these coordinates to create the claim, but we also need to make sure that the next airdrop doesn't try to create a claim with the same name. For this, we will use the `{{time}}` variable. This outputs the real-world time when the command is run, and it is unlikely that another supply drop will happen at exactly the same time on another day. We'll call our claim `DeadlyDrop{{time}}`.
 
-## The hook to create the advanced claim
+## The 'command' advanced claim
 
-Now we need another hook to listen for the output of the `w2l` from our first hook. The log line for this will be
+Now we have the coordinates and a unique name, we need to put these together to create the command claim. This will trigger our second hook when it someone enters the area. We'll use the coordinates of the crate plus and minus 20 to create a square 40 blocks by 40 blocks around it. The command the claim will execute is simply to log a keyword along with the victim's steamID for our next hook, and then delete itself. The command for this is as follows:
 
 ```
-ListEntityCustom: 1. id=447912, type=EntitySupplyCrate, name=sc_General, pos=(-125,50,-20)
-```
-
-We don't want to risk creating ambush claims on admins who are using `lce` to look for players' vehicles, so we will use a regex filter on this hook to make sure we only capture SupplyCrates.
-
-Create a logLine hook with the regex filter `ListEntityCustom.*EntitySupplyCrate` - the .* in the middle means this will match any other characters as long as both other sections of text are found around them.
-
-For this hook we need to pick up the SupplyCrate's entityID, and the coordinates again (as they are not stored from the previous hook). Add the following three custom variables:
-
-```
-SupplyCrateId : (?<=ListEntityCustom: \d\. id=)(\d{1,9})
-xcoord :        (?<=pos\=\()(-?\d{1,4})
-zcoord :        (?<=pos\=\(-?\d{1,4},-?\d{1,4},)(-?\d{1,4})
-```
-
-Now we will create a command claim using CPM which will trigger our final hook when it is entered. We'll use the coordinates of the crate plus and minus 20 to create a square 40 blocks by 40 blocks around it. The command the claim will execute is simply to log a keyword along with the victim's steamID for our final hook, and then delete itself. The command for this is as follows:
-
-```handlebars
-ccc add {{custom.SupplyCrateId}} {{subtract custom.xcoord 20}} {{subtract custom.xcoord -20}} {{subtract custom.zcoord -20}} {{subtract custom.zcoord 20}} -1 "command: w2l 'DADambush ${steamId}'; ccc remove {{custom.SupplyCrateId}}"
+ccc add DeadlyDrop{{time}} {{subtract custom.xcoord 20}} {{subtract custom.xcoord -20}} {{subtract custom.zcoord -20}} {{subtract custom.zcoord 20}} -1 "command: w2l 'DADambush ${steamId}'; ccc remove DeadlyDrop{{time}}"
```

Comment:
Just FYI, there's also an `add` helper. 

```suggestion
ccc add DeadlyDrop{{time}} {{subtract custom.xcoord 20}} {{add custom.xcoord 20}} {{add custom.zcoord 20}} {{subtract custom.zcoord 20}} -1 "command: w2l 'DADambush ${steamId}'; ccc remove DeadlyDrop{{time}}"
```

Reply:
We used `subtract` because `add` was doing string concatenation. Maybe it warrants explaining why?

Reply:
Oh right, I thought that too but then I looked at the code and it looks like I added some logic for that

https://github.com/CatalysmsServerManager/7-days-to-die-server-manager/blob/034fa6a3cf2992eef42db93bb64f416d161d2039/worker/util/Handlebars.js#L34

Is that not working?

Reply:
I'll have to test it, I'll let you know. I'm sure it's fine :)